src/components/Col.js

```diff
@@ -60,8 +60,8 @@ function getClassNames(props) {
     .concat(extraClasses);
 }
 
-export default function Col(props) {
-  const classNames = getClassNames(props);
+export function Col(props) {
```

Comment:
I see no reason for converting this default export to a named one, I see this only as a risk of breaking something. I described below how leaving this as a default export would reflect on the rest of the changes.